test/integration/setup.ts

```diff
@@ -66,30 +68,30 @@ before(() => {
   databaseUrl = 'https://' + projectId + '.firebaseio.com';
   storageBucket = projectId + '.appspot.com';
 
-  defaultApp = admin.initializeApp({
-    credential: admin.credential.cert(serviceAccount),
+  defaultApp = initializeApp({
+    credential: admin.cert(serviceAccount),
```

Comment:
Not sure if it'd be a better idea for `cert` to be exported separately ... I'll revisit this idea